### DISCUSSION TITLE
Invoice checkout window rate missing

### DISCUSSION THREAD

**Discussion by cryptokq** [btcpayserver/btcpayserver#3990] [Technical Support] [answered]
When paying an invoice in BTC, the BTC_USD rate is not shown in the checkout window, it only shows the total in BTC. How can i display the rate?

The invoice event log shows
BTC_BTC: The rating rule is 1.0

**Comment by Kukks** [comment, marked as answer]
You need to create the invoice with USD as the currency.

**Comment by cryptokq** [comment]
Paying in USD works fine, but fluctuates if wanting to pay an exact amount in BTC.
I am trying to pay with BTC as mentioned, is this the intended behavior not to show the rate when paying the invoice in BTC?

**Reply by pavlenex** [reply]
As Kukks said, you created an invoice in BTC as currency, 1 BTC = 1 BTC, so it's an intended behaviour.
